rockcraft/cli.py

```diff
@@ -54,4 +56,6 @@ def _create_app() -> "Rockcraft":
         ],
     )
 
+    app.add_command_group("Lifecycle", [appcommands.RemoteBuild])
```

Comment:
Nice to see craft-app's command group merging is working.

Reply:
yeap thanks for this! Very useful